qiskit/providers/aer/openpulse/readme.md

```diff
@@ -4,7 +4,22 @@ This simulates job using the open pulse format (see the spec).
 
 ## Example
 
-XXX
+Consider the Hamiltonian H=-w * sigmaz/2  + w1 * cos(wt+phi) * sigmax/2. The Hamiltonian may be specified as:
+```
+hamiltonian = {}
+hamiltonian['h_str'] = []
+
+#Q0 terms
+hamiltonian['h_str'].append('-0.5*w0*Z0')
+hamiltonian['h_str'].append('0.5*w1*X0||D0')
+
+hamiltonian['vars'] = {'w0': (add w0 val here), 'w1': (add w1 value here)}
+
+# set the qubit dimension to 2
+hamiltonian['qub'] = {'0': 2}
+```
+
+This particular example can be analyzed analytically using the rotating frame transformation (apply the unitary U=e^{-iwt * sigmaz/2} to the state) and the rotating wave approximation (RWA). The results obtained may be compared to the pulse simulator (see the pulse simulator test files).
```

Comment:
I don't think we necessarily need to include how this is done since it is a given for people in the field. Can just mention the dynamics of this system have a closed-form analytic expression under the RWA.